purchase_order_channel/views/purchase_order_views.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="purchase_order_form_inherit" model="ir.ui.view">
+        <field name="name">purchase.order.form.inherit</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="priority" eval="18" />
+        <field name="arch" type="xml">
+            <xpath expr="//sheet/group" position="inside">
+                <group>
+                    <field name="call_back" />
+                    <field name="request_channel_id" />
+                    <field name="request_medium_id" />
+                </group>
+            </xpath>
+        </field>
+    </record>
+    <record id="purchase_order_kpis_tree" model="ir.ui.view">
+        <field name="name">purchase.order.tree</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_kpis_tree" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='date_order']" position="before">
+                <field name="call_back" />
+                <field name="request_channel_id" />
+                <field name="request_medium_id" />
+            </xpath>
+        </field>
+    </record>
+    <record id="purchase_order_view_tree" model="ir.ui.view">
+        <field name="name">purchase.order.tree</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_view_tree" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='date_planned']" position="before">
+                <field name="call_back" />
+                <field name="request_channel_id" />
+                <field name="request_medium_id" />
+            </xpath>
+        </field>
+    </record>
+    <record id="view_purchase_order_filter" model="ir.ui.view">
+        <field name="name">request.quotation.select</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.view_purchase_order_filter" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='product_id']" position="before">
+                <field name="call_back" />
```

Comment:
What is the mean of `call_back` here?

Reply:
This field comes from sst-custom.
https://github.com/qrtl/sst-custom/blob/059197e512be7f09dd5e1b38b75c04a73df4804b/purchase_order_channel/models/purchase_order.py#L10

Reply:
Sorry my comment lacks information.  I think the `call_back` field is boolean, so it won't show in the search view.
Please let me know if I have misunderstood something.
![2023-12-18_11h22_40](https://github.com/qrtl/rmm-custom/assets/101688144/e6f2f6a5-40b0-43d0-984f-e7242579b080)

Reply:
You right. I have removed this in my latest update.